### DISCUSSION TITLE
如何在iOS使用V2ray

### DISCUSSION THREAD

**Discussion by xiejiachen** [v2fly/v2ray-core#1310] [Q&A] [answered]
尝试了许多app， 如Shadowrocket。
这些app无法单独设置outbound和inbounds以及 ws传输。
请问大家怎么做的
服务器已经配置好，使用v2rayN导入自定义配置可以正常连接

**Comment by Remonli** [comment, marked as answer]
iOS用不了，以前的版本限制NE内存不能超过15M，直接导致之前那个原生v2ray客户端的app非常不稳定，频繁被杀进程。
